app/modules/encounters/models.py

```diff
@@ -117,19 +116,39 @@ def __repr__(self):
     def get_owner(self):
         return self.owner
 
+    def get_owner_guid_str(self):
+        return str(self.owner.guid)
+
     def get_sighting(self):
         return self.sighting
 
-    def get_location(self):
-        edm_data = current_app.edm.get_dict('encounter.data_complete', self.guid)
-        location_id = None
-        if isinstance(edm_data, dict) and edm_data.get('success', False):
-            edm_json = edm_data['result']
+    def get_sighting_guid_str(self):
+        return str(self.sighting.guid)
 
-            if 'locationId' in edm_json.keys():
-                location_id = edm_json['locationId']
+    # first tries encounter.locationId, but will use sighting.locationId if none on encounter,
+    #   unless sighting_fallback=False
+    def get_location_id(self, sighting_fallback=True):
+        location_id = self.get_edm_data_field('locationId')
+        if not location_id and sighting_fallback and self.sighting:
+            location_id = self.sighting.get_location_id()
         return location_id
 
+    def get_location(self):
+        return self.get_edm_data_field('locationId')
+
+    def get_point(self):
+        lat = self.get_edm_data_field('decimalLatitude')
+        long = self.get_edm_data_field('decimalLongitude')
+        return f'{lat},{long}' if lat and long else None
```

Comment:
This is a big guess about what is m eant by 'point'

Reply:
my reading of [these docs](https://www.elastic.co/guide/en/elasticsearch/reference/current/geo-point.html) is that this is a valid format.  yet: these docs seem to swap (!!!) lat/lon for lon/lat on some of these formats.  what???   the string-version though appears to be `lat,lon`.   (unlike the _array_ and _WKT POINT_ for some reason!)